docs/tendermint-core/state-sync.md

```diff
@@ -0,0 +1,49 @@
+--- 
+order: 10
+---
+
+# State Sync
+
+With fast sync a node is downloading all of the data of an application from genesis and verifying it. 
+With state sync your node will download data related to the head or near the head of the chain and verify the data. 
+This leads to drastically shorter times for joining a network. 
+
+## Using State Sync
+
+> NOTE: Before trying to use state sync, see if the application you are operating a node for supports it. 
+
+Under the state sync section in `config.toml` you will find multiple settings that need to be configured in order for your node to use state sync.
+
+Lets breakdown the settings:
+
+- `enable`: Enable is to inform the node that you will be using state sync to bootstrap your node.
+- `rpc_servers`: RPC servers are needed because state sync utilizes the light client for verification. 
+    - It is recommended to have more than 2 servers. 
+- `temp_dir`: Temporary directory is store the chunks in the machines local storage, If nothing is set it will create a directory in `/tmp`
+
+The next information you will need to acquire it through publicly exposed RPC's or a block explorer which you trust. 
+
+- `trust_height`: Trust height is needed to inform the light client of your trusted height. 
+- `trust_hash` Trust hash is needed to inform the light client of your trusted hash. 
```

Comment:
We should clarify these. `trust_hash` is the block hash of a trusted block (from `block_id.hash` in the header), and `trust_height` is the height of the trusted block.

Reply:
Could you not get the hash from multiple places in the RPC. I added an example below on how to get it. Should there be more info in the line for where to get it?

Reply:
Nah, the example is fine. But the descriptions of these options are a little too vague: they're meant to explain what a trusted height and hash are, these just say that they should be the trusted height and hash, so they don't explain anything.